### ISSUE TITLE
release code for baselines

### ISSUE THREAD

**Issue by sean-metzger** [msn199959/Logo-2k-plus-Dataset#9]
Hello,

Could you please release the code for how you trained the baselines (esp. resnet-50)? Training a resnet-50 (pretrained on imagenet) gives only 13% accuracy for me. Did you train one from scratch for what's in the paper?
